src/components/value-list/utils.ts

```diff
@@ -0,0 +1,63 @@
+import { getItemIndex } from "../pick-list/shared-list-logic";
+import { DragStatus } from "./interfaces";
+import { ValueList } from "./value-list";
+
+export function getScreenReaderText(
+  item: HTMLCalciteValueListItemElement,
+  status: DragStatus,
+  valueList: ValueList<HTMLCalciteValueListItemElement>
+): string {
+  const {
+    items,
+    intlDragHandleStart,
+    intlDragHandleActivated,
+    intlDragHandleNewPosition,
+    intlDragHandleCurrentPosition
+  } = valueList;
+
+  const total = items.length;
+  const position = getItemIndex(valueList, item) + 1;
+
+  if (status === "idle") {
+    const idleText = intlDragHandleStart
+      ? intlDragHandleStart
```

Comment:
You'll have to do a manual replacement for the placeholder tokens on user-provided strings, otherwise they won't get replaced (template literals are interpolated at creation-time). We'll be working on built-in translations soon, so this can be a simple string replacement.